Canvas: Element level data links (#89079)
grafana/grafana: docs/sources/panels-visualizations/configure-data-links/index.md

1. Click **Save** to save changes and close the dialog box.
1. Click **Save dashboard**.
1. Click **Back to dashboard** and then **Exit edit**.

> {{% admonition type="note" %}}
> You can rearrange your data links to determine the order they appear in the visualization. To do this, click and drag the data link to the desired position.
> {{% /admonition %}}

Reviewer: ```suggestion
If you add multiple data links, you can control the order in which they appear in the visualization. To do this, click and drag the data link to the desired position.
```
